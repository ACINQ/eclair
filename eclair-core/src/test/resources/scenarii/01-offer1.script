# Simple test that we can commit an HTLC
# Initial state: A=1000000 sat, B=1000000 sat, both fee rates=10000 sat
A:offer 1000000,9f86d081884c7d659a2feaa0c55ad015a3bf4f1b2b0b822cd15d6c15b0f00a08
B:recvoffer
A:commit
B:recvcommit
A:recvrevoke
B:commit
A:recvcommit
B:recvrevoke
checksync
echo ***A***
A:dump
echo ***B***
B:dump
