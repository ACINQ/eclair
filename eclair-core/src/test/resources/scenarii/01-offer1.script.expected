***A***
LOCAL COMMITS:
 Commit 1:
  Offered htlcs: (0,1000000 msat)
  Received htlcs:
  Balance us: 999000000 msat
  Balance them: 1000000000 msat
  Fee rate: 10000
REMOTE COMMITS:
 Commit 1:
  Offered htlcs:
  Received htlcs: (0,1000000 msat)
  Balance us: 1000000000 msat
  Balance them: 999000000 msat
  Fee rate: 10000
***B***
LOCAL COMMITS:
 Commit 1:
  Offered htlcs:
  Received htlcs: (0,1000000 msat)
  Balance us: 1000000000 msat
  Balance them: 999000000 msat
  Fee rate: 10000
REMOTE COMMITS:
 Commit 1:
  Offered htlcs: (0,1000000 msat)
  Received htlcs:
  Balance us: 999000000 msat
  Balance them: 1000000000 msat
  Fee rate: 10000
